spec/rester/rspec_spec.rb

```diff
@@ -4,20 +4,63 @@
   describe '/v1/tests' do
     context 'GET' do
       context 'With some context' do
-        it { is_expected.to eq stub_response }
-        it { is_expected.to eq [200, '{"string":"some_string","integer":1,"float":1.23,"symbol":"some_symbol","bool":true,"method":"search"}'] }
+        it { is_expected.to include stub_response }
+        it {
+          is_expected.to include(
+            "string" => "some_string",
+            "integer" => 1,
+            "float" => 1.23,
+            "symbol" => "some_symbol",
+            "bool" => true,
+            "method" => "search"
+          )
+        }
+      end
```

Comment:
Once we move to the `response[successful=false]` syntax, I think we'll pass the adapter to an instance of `Rester::Client` and call `request(verb, path,...` on it instead calling it on the adapter directly.  This way we won't need to worry about duplicating this parsing logic since the client will handle it for us.  And then we'll just check that the response from the client was `successful?` or not based on what's in the response tags.